src/Workspaces/Core/Portable/Workspace/PrimaryWorkspace.cs

```diff
@@ -1,22 +1,27 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
+using System.Composition;
 using System.Threading;
-using System.Threading.Tasks;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis
 {
-    internal static class PrimaryWorkspace
+    [Export(typeof(PrimaryWorkspace)), Shared]
+    internal sealed class PrimaryWorkspace
     {
-        private static readonly ReaderWriterLockSlim s_registryGate = new ReaderWriterLockSlim(LockRecursionPolicy.NoRecursion);
-        private static Workspace s_primaryWorkspace;
+        private readonly ReaderWriterLockSlim s_registryGate = new ReaderWriterLockSlim(LockRecursionPolicy.NoRecursion);
```

Comment:
>s_registryGate [](start = 46, length = 14)

s_ prefix no longer needed as this is an instance variable.

Reply:
:arrow_right: Fixed in 9c28f4a